ModelCatalogueGenomicsPlugin/grails-app/controllers/org/modelcatalogue/core/genomics/GenomicsController.groovy

```diff
@@ -1,12 +1,71 @@
 package org.modelcatalogue.core.genomics
 
+import groovy.util.logging.Log4j
+import org.modelcatalogue.core.AbstractCatalogueElementController
+import org.modelcatalogue.core.Asset
 import org.modelcatalogue.core.DataClass
+import org.modelcatalogue.core.api.ElementStatus
 import org.springframework.http.HttpStatus
 
 /**
  * Controller for GEL specific reports.
  */
-class GenomicsController {
+@Log4j
+class GenomicsController extends AbstractCatalogueElementController<DataClass> {
```

Comment:
there is no reason extending `AbstractCatalogueElementController` unless you want to create endpoint for the domain class.